cpp/tests/CMakeLists.txt

```diff
@@ -2,12 +2,18 @@
 # Now simply link against gtest or gtest_main as needed. Eg
 set(TEST_SOURCES
   gtest_main.cpp
-  # constraints.cpp
+  constraints.cpp
```

Comment:
Please append test_ prefix to test source files.

Reply:
Ok.